### PR TITLE
MCOL-641 Cleanup.

### PR DESCRIPTION
**Pull request by tntnatbry**

### CODE REVIEW

File: utils/thrift/thrift/protocol/TCompactProtocol.tcc [hunk 1]
```diff
@@ -808,8 +808,6 @@ TType TCompactProtocolT<Transport_>::getTType(int8_t type) {
     case detail::compact::CT_STRUCT:
       return T_STRUCT;
     default:
-
-      cout << __FILE__<< __LINE__ << __func__<< endl;
```
Comment (drrtuy) [inline utils/thrift/thrift/protocol/TCompactProtocol.tcc:812]: Are you sure this relates to MCOL-641?

File: dbcon/joblist/tuple-bps.cpp [hunk 2]
```diff
@@ -1512,7 +1512,7 @@ bool TupleBPS::processSingleFilterString(int8_t BOP, int8_t colWidth, T val, con
                 break;
 
             case 16:
-                bigVal2 = *((int128_t*) filterString);
+                datatypes::TSInt128::assignPtrPtr(&bigVal2, (int128_t*) filterString);
```
Comment (drrtuy) [inline dbcon/joblist/tuple-bps.cpp]: I suggest to make bigVal2 datatypes::TSInt128 and replace assignPtrPtr here with bigVal2 = datatypes::TSInt128(reinterpret_cast<int128_t*>(filterString)). Then compareSingleValue() will have bigVal2.getValue() as the last argument at the line 1527.

File: dbcon/joblist/rowestimator.cpp [hunk 3]
```diff
@@ -380,7 +380,8 @@ float RowEstimator::estimateRowReturnFactor(const BRM::EMEntry& emEntry,
                     if (ct.colDataType == execplan::CalpontSystemCatalog::DECIMAL ||
                         ct.colDataType == execplan::CalpontSystemCatalog::UDECIMAL)
                     {
-                        uint128_t val = *(uint128_t*)msgDataPtr;
+                        uint128_t val;
```
Comment (drrtuy) [inline dbcon/joblist/rowestimator.cpp]: Plz take a look at the suggestion bellow in the tuple-bps.cpp regarding datatypes::TSInt128.

**Review by drrtuy — CHANGES_REQUESTED** [review]
There are number of WIP acronyms related to the MCOL-641. Could you plz replace them with TODO ?

File: utils/thrift/thrift/protocol/TCompactProtocol.tcc (hunk 1 above)
Comment (tntnatbry) [reply utils/thrift/thrift/protocol/TCompactProtocol.tcc:812]: Yes, this got added by mistake in the big MCOL-641 PR: https://github.com/mariadb-corporation/mariadb-columnstore-engine/pull/1604

File: dbcon/joblist/tuple-bps.cpp (hunk 2 above)
Comment (tntnatbry) [reply dbcon/joblist/tuple-bps.cpp]: Thanks for this suggestion, I have updated the code.

File: dbcon/joblist/rowestimator.cpp (hunk 3 above)
Comment (tntnatbry) [reply dbcon/joblist/rowestimator.cpp]: Done.

**Comment by tntnatbry** [comment]
@drrtuy PR is updated with the suggested changes, please review.

File: datatypes/mcs_int128.h [hunk 4]
```diff
@@ -265,6 +265,13 @@ class TSInt128
       return toTFloat128();
     }
 
+    //    unaligned argument
+    inline TSInt128& operator=(const int128_t* x)
```
Comment (drrtuy) [inline datatypes/mcs_int128.h:269]: This assignment operator looks a bit odd IMHO but TSInt128 is our class and we can have whatever semantics we need.

**Review by drrtuy — APPROVED** [review]

**Pull request merged**